xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -451,10 +559,19 @@
         <Parameter Name="value" Type="System.Byte" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to reverse.</param>
+        <summary>Reverses a primitive value by performing an endianness swap of the specified <see cref="T:System.Byte" /> value, which effectively does nothing for a <see cref="T:System.Byte" />.</summary>
+        <returns>The passed-in value, unmodified.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method allows reading a struct of numeric primitives and reverse each field within it without having to special case <see cref="T:System.Byte" /> fields.
+This method is a no-op and added only for consistency.
```

Comment:
Do we need to add a line break between the two remarks for it to render as expected?

Similar to https://github.com/dotnet/dotnet-api-docs/blob/62930dbca63d10c87b40945e20f5a06acd98a413/xml/System.Text.Json/Utf8JsonWriter.xml#L108-L113

Reply:
With no line break, it behaves like it's one paragraph. It depends on the desired intention.

In this case, I think it should have a line break or even remove the first sentence since it's a no-op?

Reply:
> or even remove the first sentence since it's a no-op?

I would be fine with removing the first sentence. Seems to highlight a specific usage scenario of the API, but isn't applicable generally.

Most readers probably wouldn't care that this API could be used in this way.

So, remove:
```text
This method allows reading a struct of numeric primitives and reverse each field within it without having to special case <see cref="T:System.Byte" /> fields.
```

Reply:
nit: I recall previously, there was a question of using "no-op". It isn't super user friendly, so we could rephrase it/add some more context (though no-op is a term used in low level computing - https://en.wikipedia.org/wiki/NOP_(code))

I am OK with leaving it as is too.

```suggestion
This method effectively does nothing (a no-op) and added only for consistency.
```

Reply:
Yep, we just merged a PR from @bartonjs I believed where we changed this. I'll try to find the wording we used.